src/rgw/rgw_amqp.cc

```diff
@@ -836,8 +836,11 @@ class Manager {
       // when a new connection is added.
       connections.max_load_factor(10.0);
       // give the runner thread a name for easier debugging
-      const auto rc = ceph_pthread_setname(runner.native_handle(), "amqp_manager");
-      ceph_assert(rc==0);
+      const std::string thread_name = "amqp_manager";
```

Comment:
might as well use `const char*` for string literals so we don't have to copy them into a std::string and call c_str() on it